cmd/verrazzano-operator/main.go

```diff
@@ -35,6 +35,7 @@ var (
 	helidonAppOperatorDeployment string
 	enableMonitoringStorage      string
 	apiServerRealm               string
+	imagePullSecret              string
```

Comment:
generic comment - since we are in here doing this, should we make it accept a list?

Reply:
I agree, I was looking at making it a list.  But the startup args for the containers are strings.  I could embed some array syntax into a string, but that didn't feel clean.

Another approach I thought of that I could look into instead is to have the verrazzano-operator read the imagePullSecret list from the verrazzano-operator service account.  And then add that list to each service account created by verrazzano-operator.